### ISSUE TITLE
Is it possible to use cluster server only?

### ISSUE THREAD

**Issue by artpolikarpov** [meteorhacks/cluster#63]
I want to connect to microservices on the serverside. But I don’t want to let cluster change my ddp urls on the client, because I have problems with it (cluster’s stikyness isn’t working when app is in iframe in Safari, because of the cookie policy). I already solved this problem, so I just don’t want any package touching my `DDP_DEFAULT_CONNECTION_URL` on my public instances.

But I do want to try cluster for MicroServices. And for multicore support. Is it possible?

**Comment by arunoda** [comment]
Cluster does load balancing using changing these URLs. So, you got no luck here.
But you can use multi-core support. Then, only expose `CLUSTER_WORKERS_COUNT` env var.

**Comment by artpolikarpov** [comment]
Good news about multi-core support, thank you.

Let me try again to explore what I want for MicroServices. Look, I don’t want to use Cluster for load balancing on “main” servers. And I don’t want this “main” servers to register like a service in cluster terminology. But I want to connect to another microservices from them using Cluster.
